prov/efa/src/rdm/rxr_env.c

```diff
@@ -45,7 +45,7 @@ struct rxr_env rxr_env = {
 	.use_zcpy_rx = 1,
 	.set_cuda_sync_memops = 1,
 	.zcpy_rx_seed = 0,
-	.shm_av_size = 128,
+	.shm_av_size = 256,
```

Comment:
Clarify - If the user set `shm_av_size` to 256 on a small instance, will that cause any issue, i.e. oversubscription? Do we need to defend against that?

Reply:
>  Clarify - If the user set shm_av_size to 256 on a small instance, will that cause any issue, i.e. oversubscription? Do we need to defend against that?

It should out. This handle the case that larger number of processes are used. It does not cause more processes to be used.